### ISSUE TITLE
Add a repo on the home page?

### ISSUE THREAD

**Issue by devth** [yetibot/yetibot.github.io#6]

**Comment by cvic** [comment]
This should work like the live docs?

**Comment by devth** [comment]
Yeah except allow the user to input commands, hit enter, see the result, like a normal REPL.

Haven't found a good lib to do that (might have to write one).
